src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -51,6 +51,8 @@
 
 from .tunnel import TunnelServer
 
+from .utils import (_get_location_from_resource_group, _list_app, _rename_server_farm_props, _get_location_from_webapp)
```

Comment:
nit: There are already functions imported from `.utils`, combine this line with below